File: AndroidTemplate/app/src/main/java/com/corp/k/androidos/android/TempApplication.java
package com.corp.k.androidos.android;

import android.app.Application;
import android.content.Context;
import android.graphics.Point;

public class TempApplication extends Application {
    private String TAG = TempApplication.class.getSimpleName();
    private static Point screenSize;
    private static TempApplication sInstance;

    @Override
    public void onCreate() {
        super.onCreate();
        sInstance = this;
    }

    public static Context getAppContext() {
        return sInstance.getApplicationContext();
    }

    public static TempApplication getInstance(){
        return sInstance;
    }

    public static void setScreenSize(Point size){
        screenSize = size;
    }
    public static Point getScreenSize(){
        return screenSize;
    }
}

File: AndroidTemplate/app/src/main/java/com/corp/k/androidos/kotlin/TempApplication.kt
package com.corp.k.androidos.kotlin

import android.app.Application
import android.content.Context

/**
 * Created by hoangtuan on 8/14/17.
 */
open class TempApplication : Application() {
    init {
        instance = this
    }

    companion object {
        private var instance: TempApplication? = null

        fun applicationContext() : Context {
            return instance!!.applicationContext
        }
    }

    override fun onCreate() {
        super.onCreate()
        // initialize for any

        // Use ApplicationContext.
        // example: SharedPreferences etc...
        val context: Context = applicationContext()
    }
}